### PR TITLE
Removed paas related code

### PR DESCRIPTION
**Pull request by VenkataChalla554**
### Trello card
https://trello.com/c/gmRLlM2q/842-code-clean-up-teacher-training-advisor
### Context
Removed unused codes
### Changes proposed in this pull request
Removed terraform code and ruby code.
### Guidance to review
Terraform code , ruby code removed and updated reade.me

### CODE REVIEW

File: README.md
Comment (saliceti) [inline README.md:1]: Please replace with a simple README like: https://github.com/DFE-Digital/cpd-information/blob/main/README.md
It redirects to getintoteaching.education.gov.uk

**Comment by saliceti** [comment]
Also please remove the terraform/.gitignore file and replace with .gitignore at the root like: https://github.com/DFE-Digital/cpd-information/blob/main/.gitignore

File: README.md
Comment (VenkataChalla554) [reply README.md:1]: Updated readme as suggested

File: bin/.gitkeep
[the remark this answers is not in the document]
Comment (VenkataChalla554) [reply bin/.gitkeep:1]: Fixed this

**Review by saliceti — APPROVED** [review]

**Pull request merged**